<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BallsInRows</class>
 <widget class="QMainWindow" name="BallsInRows">
  <property name="windowTitle">
   <string>Balls in Rows</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="winnerScoreLabel">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gamerScoreLabel">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="winnerNameLabel">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gamerNameLabel">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="newLabel">
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colorsLabel">
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Colors</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;Pause</string>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu">
   <widget class="QMenu" name="gameMenu">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="newGameAction"/>
    <addaction name="openGameAction"/>
    <addaction name="saveGameAction"/>
    <addaction name="abortGameAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="scoreMenu">
    <property name="title">
     <string>&amp;Score</string>
    </property>
    <addaction name="highscoresAction"/>
    <addaction name="picturesAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="fullscreenAction"/>
    <addaction name="changeNameAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="gameMenu"/>
   <addaction name="scoreMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="newGameAction">
   <property name="text">
    <string>&amp;New game</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="openGameAction">
   <property name="text">
    <string>&amp;Continue saved game</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="saveGameAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save game</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="abortGameAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Abort game</string>
   </property>
   <property name="shortcut">
    <string notr="true">Esc</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="highscoresAction">
   <property name="text">
    <string>&amp;Highscores</string>
   </property>
  </action>
  <action name="picturesAction">
   <property name="text">
    <string>&amp;Pictures</string>
   </property>
  </action>
  <action name="fullscreenAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string notr="true">F11</string>
   </property>
  </action>
  <action name="changeNameAction">
   <property name="text">
    <string>&amp;Change gamer name</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>BallsInRows</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
